tests/Validation/ValidationValidatorTest.php

```diff
@@ -5947,7 +5947,7 @@ public function testValidateTimezoneWithAllWithBCOption()
         $v = new Validator($trans, ['foo' => 'GMT'], ['foo' => 'Timezone:All_with_BC']);
         $this->assertTrue($v->passes());
 
-        $v = new Validator($trans, ['foo' => 'GB'], ['foo' => 'Timezone:All_with_BC']);
+        $v = new Validator($trans, ['foo' => 'Europe/London'], ['foo' => 'Timezone:All_with_BC']);
```

Comment:
The test is to cover `Timezone:All_with_BC`, is there any other alternative `GB` that's still working as a backward compatible example: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones